pkg/scheduler/flavorassigner/tas_flavorassigner.go

```diff
@@ -29,15 +29,17 @@ import (
 
 // WorkloadsTopologyRequests - returns the TopologyRequests of the workload
 func (a *Assignment) WorkloadsTopologyRequests(wl *workload.Info, cq *cache.ClusterQueueSnapshot) cache.WorkloadTASRequests {
+	isTASOnly := cq.IsTASOnly()
 	tasRequests := make(cache.WorkloadTASRequests)
 	for i, podSet := range wl.Obj.Spec.PodSets {
-		if podSet.TopologyRequest != nil {
+		if podSet.TopologyRequest != nil || isTASOnly {
 			psAssignment := a.podSetAssignmentByName(podSet.Name)
 			if psAssignment.Status.IsError() {
 				// There is no resource quota assignment for the PodSet - no need to check TAS.
 				continue
 			}
-			psTASRequest, err := podSetTopologyRequest(psAssignment, wl, cq, i)
+			isTASImplied := podSet.TopologyRequest == nil && isTASOnly
```

Comment:
It looks like there are some places to check `podSet.TopologyRequest == nil && isTASOnly`.
So, could we prepare a dedicated function to check if the request implies TAS?